util/requirements2.txt

```diff
@@ -4,4 +4,9 @@ jinja2   # A dependency for the mbed build library
 colorama # A dependency for the mbed build library
 cpp-coveralls # A dependency for coverage testing
 
-pyOCD # for debugging code on MBED
+# firmware - required
+mbed-ls
```

Comment:
Until they merge your PR, you can change this to:

~~~{.sh}
git+https://github.com/jjones646/mbed-ls@issue_38_squashed
~~~